framework/scripts/restart_drill_script

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+source ./conf/drillTestConfig.properties
+set -x
+clush -a ${DRILL_HOME}/bin/drillbit.sh restart
```

Comment:
I understand. Thanks. Let's add more comments in here or in the template. When we actively use this script / restart feature, we can work on enhancing it. We should have option to start, shutdown, and restart. Right now I'm okay getting this in as-is.